.gitignore

```diff
@@ -1,2 +1,3 @@
-vendor
-composer.lock
+/vendor/
+/composer.lock
```

Comment:
It appears to me no reason this project would be installed as a local dependency.
However, you made a point with tightly bounded constraints, in case this project is installed globally, as people can do with `phpunit`, `psysh`, `cs-fixer`, ...

Reviewing the current constraints, not locking the deps and configuring the Travis build matrix in order to test lowest and highest dep constraints would help ensuring that, if we're actually going this way. 
But is it actually worth it ? I don't think so, because the project is mainly thought to be used through the `.phar` as a standalone application, with its own dep constraints, and so... :trollface: => 🚪 

(This debate is endless 😸 )